Tasks/ServiceFabricPowerShell/ServiceFabricPowerShell.ps1

```diff
@@ -0,0 +1,90 @@
+Trace-VstsEnteringInvocation $MyInvocation
```

Comment:
Somewhere in this file, it might be worthwhile to comment that it follows the same pattern as the AzurePowerShell task.